test-harness-core/src/main/java/edu/umd/cs/findbugs/test/AnalysisRunner.java

```diff
@@ -110,8 +110,7 @@ public BugCollectionBugReporter run(Path... files) {
             if (!bugReporter.getQueuedErrors().isEmpty()) {
                 AssertionError assertionError = new AssertionError(
```

Comment:
No need for a local variable any more, just create and throw it after reporting bugs.